deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-role.yaml

```diff
@@ -65,7 +68,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
```

Comment:
Do we need a reference to kubernetes specification for every kind ? With a comment for a full explanation of Role parameters check here ?

Reply:
> Do we need a reference to kubernetes specification for every kind ?

I think it is a generic information, not related to the elastic-agent (like variables, why some paths are mounted, why some parameters are enabled in context of elastic-agent functionality)

> With a comment for a full explanation of Role parameters check here ?

what do you mean by `Role parameters check` ?

Reply:
Like an example comment pointing here: https://kubernetes.io/docs/reference/access-authn-authz/rbac/

Yes not convinced if we need such general links. Just mentioning this here in any case in order others to comment as well

Reply:
I believe that we shouldn't add extra info here. Users should be familiar with rbac. Just some comments for which purpose some resources are needed because users may wonder why agent needs so many permissions after all. What I am not sure about is if those resources required by cloud beat should be enabled by default or not...